tests/rhtap-demo/rhtap-demo.go

```diff
@@ -51,7 +51,7 @@ const (
 	// Timeouts
 	appDeployTimeout            = time.Minute * 20
 	appRouteAvailableTimeout    = time.Minute * 5
-	customResourceUpdateTimeout = time.Minute * 5
```

Comment:
let's change it to 10mins and see. We don't want to wait for additional minutes in case of any failures

Reply:
deal,
done